### DISCUSSION TITLE
Read information + Add Button

### DISCUSSION THREAD

**Discussion by ghost** [jocmp/capyreader#270] [Feature Requests]
It would be nice if the screen remained blank like in the Feedly app when you read an article in a publication. Also, it would be a little more useful if the RSS add button was to the right of the mark all read button in the top right bar.

**Comment by jocmp** [comment]
@ismailsevik Can you share a screenshot of what you mean by blank screen? I downloaded Feedly to look and I don't think I'm seeing the same.

**Reply by ghost** [reply]
![Screenshot_20240816_003746_Feedly](https://github.com/user-attachments/assets/cbaf87ad-9ec8-47d6-9b72-b80f76af028b)
Like this. Normally, only read articles should not be listed.

**Comment by ghost** [comment]
![Screenshot_20240816_004130_Capy Reader](https://github.com/user-attachments/assets/9f442f3a-5728-4091-965b-0f10891185f6)
It shouldn't be like this image.

**Comment by jocmp** [comment]
Thanks for the screenshots. I see what you mean now. This is based on Feedbin's implementation which shows you all articles on the "All" filter even if they're read. 

<img src="https://github.com/user-attachments/assets/0dfd09f4-df61-49a8-b2dd-df8dc335b313" width="400px">

I want to keep it like this because it provides a better experience in widescreen where it's important to keep the selection of the current article while it's being read

<img src="https://github.com/user-attachments/assets/cc37f336-3cb1-49ab-873f-2c83566143f2">

Articles will disappear when you have the "Unread" filter selected. I know that's different from Feedly's implementation, but it's meant to be that way in this app.

**Reply by ghost** [reply]
The add RSS button is not on the article page. It would be nice if it was only on the home page. I mean the add RSS feed button on the right of the 2nd image where there are three dots. Happy developing though. :)
